libafl/src/bolts/os/windows_exceptions.rs

```diff
@@ -150,6 +152,7 @@ pub static CRASH_EXCEPTIONS: &[ExceptionCode] = &[
     ExceptionCode::HeapCorruption,
     ExceptionCode::StackBufferOverrun,
     ExceptionCode::AssertionFailure,
+    ExceptionCode::Others,
```

Comment:
Singular `Other`? It's only one exception at that time, right?

Reply:
yes I added #[num_enum(default)] to ExceptionCode::Others so that if the exception is not listed above (like random exceptions raised by User's code), that exception is catecorized as `Others`

Reply:
No, I mean using singular here instead of plural, since it's a single exception being thrown

Reply:
ok. my english grammar's problem 😿